### PR TITLE
[enh] settings.yml: implement general.enable_metrics

### LINKED ISSUE #657: How to disable stats of SearXNG in docker ?
Hi,
i would like to disable the stats of my SearXNG instance using docker since it's set by default.
Thanks !

### PR DESCRIPTION
**Pull request by dalf**
## What does this PR do?

* disable recording of the statistics.
* this commit doesn't change the UI: `/stats`, `/stats/errors` return no statistics

## How to test this PR locally?

* set `global.enable_metrics: false` in settings.yml
* `make run`
* make some requests
* check there is no statistics in /stats and /stats/errors

## Related issues

* Close #657 
* https://github.com/searx/searx/commit/9b5415ea2f68b05a9c17d335ef31001df3998355

### CODE REVIEW

File: searx/settings_defaults.py [hunk 1]
```diff
@@ -142,6 +142,7 @@ def apply_schema(settings, schema, path_list):
         'debug': SettingsValue(bool, False, 'SEARXNG_DEBUG'),
         'instance_name': SettingsValue(str, 'SearXNG'),
         'contact_url': SettingsValue((None, False, str), None),
+        'enable_metrics': SettingsValue(bool, True),
```
Comment (return42) [inline searx/settings_defaults.py:145]: I miss a documentation in the [admin/engines/settings/general](https://docs.searxng.org/admin/engines/settings.html#general) section.

File: searx/settings.yml [hunk 2]
```diff
@@ -2,6 +2,7 @@ general:
   debug: false              # Debug mode, only for development
   instance_name: "SearXNG"  # displayed name
   contact_url: false        # mailto:contact@example.com
+  enable_metrics: false        # record stats
```
Comment (return42) [inline searx/settings.yml]: The old behaviour was _true_ / "True" is also set as default in the settings_defaults.py

```diff
@@ -2,7 +2,7 @@ general:
   debug: false              # Debug mode, only for development
   instance_name: "SearXNG"  # displayed name
   contact_url: false        # mailto:contact@example.com
-  enable_metrics: false        # record stats
+  enable_metrics: true      # record stats
```

**Review by return42 — APPROVED** [review]
Over all this PR LGTM .. thanks .. there a only some nits & a missing documentation .. see my comments.

File: searx/settings_defaults.py (hunk 1 above)
Comment (dalf) [reply searx/settings_defaults.py:145]: Done

File: searx/settings.yml (hunk 2 above)
Comment (dalf) [reply searx/settings.yml]: Fixed

**Review by return42 — APPROVED** [review]
Thanks for the fix.

**Pull request merged**